model/distro/distro.go

```diff
@@ -265,8 +265,6 @@ const (
 
 	CloneMethodLegacySSH = "legacy-ssh"
 	CloneMethodOAuth     = "oauth"
-
-	unconfiguredAmi = "ami-1234"
```

Comment:
it looks to me like there are still a bunch of distros in staging with the dummy AMI. Do we need to clean those up before making these changes?

Reply:
It didn't seem worth all the button clicking haha. It would only matter in staging if someone tried to spin up a host in the availability zone with the dummy AMI (the host just wouldn't come up). Since no one in prod will be creating a host in those availability zones for those distros, it's not something we'd really need to do either.

Reply:
I'm concerned about build's script: why isn't it updating the configuration in staging?

Reply:
Oh I see the confusion: there were also dummy AMIs in prod, because when we ran the original script we just updated every distro. Now build has finished writing their scripts, and said that the distros that still have dummy AMIs won't be supported in those regions, so I removed those region configurations from prod by hand.

Reply:
The regions that _are_ supported are updated by the build scripts in staging and in prod.